TargetLibraries/Snitch/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+file(GLOB_RECURSE SOURCES
+  "src/**"
+)
+
+include(cmake/snitch-runtime-precompiled.cmake)
+
+add_deeploy_library(deeploysnitch STATIC ${SOURCES})
+target_include_directories(deeploysnitch
+  PUBLIC
+  ${CMAKE_CURRENT_LIST_DIR}/inc
+)
+
+target_compile_options(deeploysnitch PRIVATE
+  -Wno-implicit-function-declaration
+  -Wno-implicit-int-conversion
+  -Wno-sign-conversion
+  -Wno-sign-compare
+)
+
+target_include_directories(deeploysnitch SYSTEM PUBLIC ${SNITCH_RUNTIME_INCLUDE})
+target_compile_options(deeploysnitch PUBLIC ${SNITCH_RUNTIME_COMPILE_FLAGS})
+
+target_link_libraries(deeploysnitch INTERFACE snitch-runtime)
+# target_sources(deeploysnitch INTERFACE $<TARGET_OBJECTS:snitch-runtime>)
```

Comment:
why did you keep this commented line?

Reply:
here https://github.com/pulp-platform/Deeploy/pull/14/commits/6bd719e46d663908db5f9652562a2d914f6948e3